examples/undocumented/python/classifier_multilabeloutputliblinear.py

```diff
@@ -7,20 +7,21 @@
 
 def classifier_multilabeloutputliblinear (fm_train_real=traindat,fm_test_real=testdat,label_train_multiclass=label_traindat,label_test_multiclass=label_testdat,width=2.1,C=1,epsilon=1e-5):
 	from shogun import RealFeatures, MulticlassLabels, MultilabelLabels
-	from shogun import MulticlassLibLinear
-
+	import shogun as sg
+	
 	feats_train=RealFeatures(fm_train_real)
 	feats_test=RealFeatures(fm_test_real)
 
 	labels=MulticlassLabels(label_train_multiclass)
 
-	classifier = MulticlassLibLinear(C,feats_train,labels)
-	classifier.train()
+	classifier = sg.machine("MulticlassLibLinear", C1=C, C2=C, labels=labels)
+	classifier.train(feats_train)
 
-	label_pred = classifier.apply_multilabel_output(feats_test,2)
-	out = label_pred.get_labels()
+	# TODO: figure out the new style API for the below call, disabling for now
```

Comment:
@lisitsyn pls comment on API suggestions for this part, as your wrote the thing IIRC :)